logic/src/commonMain/kotlin/com/wire/kalium/logic/data/session/SessionRepository.kt

```diff
@@ -205,4 +213,13 @@ internal class SessionDataSource(
     override suspend fun cookieLabel(userId: UserId): Either<StorageFailure, String?> = wrapStorageNullableRequest {
         authTokenStorage.getToken(userId.toDao())?.cookieLabel
     }
+
+    override suspend fun isAccountReadOnly(userId: UserId): Either<StorageFailure, Boolean> = wrapStorageRequest {
+        accountsDAO.getAccountManagedBy(userId.toDao()) != ManagedByEntity.WIRE
```

Comment:
afaik managed by is nullable. So are we sure in case of null then it is not managed my wire?

Reply:
Yup, good point... now I did the parity accordingly to the old scala code :D